js/player2.js

```diff
@@ -0,0 +1,114 @@
+// var currentKey2;
```

Comment:
Don't include files in the file system that you aren't using as part of this delivered version.